src/main/java/ch/njol/skript/effects/EffArmorStandExtremities.java

```diff
@@ -0,0 +1,84 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Keywords;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.ArmorStand;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Armor Stand - Extremities")
+@Description("Allows users to modify the extremities of an armor stand (i.e. whether its has arms or a base plate).")
+@Examples({
+	"show {_armorstand}'s arms",
+	"hide the base plate of {_armorstands::*}"
+})
+@Since("INSERT VERSION")
+@Keywords({"arms", "base plate"})
+public class EffArmorStandExtremities extends Effect {
+
+	static {
+		Skript.registerEffect(EffArmorStandExtremities.class,
+			"(:show|hide) %livingentities%'[s] (base[ ]plate|:arms)",
+			"(:show|hide) [the] (base[ ]plate|:arms) of %livingentities%"
```

Comment:
```suggestion
			"(:show|hide) %livingentities%'[s] (base[ ]plate|:arms)",
			"(:show|hide) [the] (base[ ]plate|:arms) of %livingentities%"
```
would be good to support `reveal` as an option